test/regexp/ferguson/bug-16891.chpl

```diff
@@ -0,0 +1,12 @@
+use IO;
```

Comment:
Do we need a skipif if no re2?

Reply:
yes, thanks

Reply:
oh, wait, no we don't, because `test/regexp/ferguson` already has a directory `SKIPIF`